qml/main.qml

```diff
@@ -39,6 +39,11 @@ ApplicationWindow {
         var tag = GitTag === "" ? "No tags!" : GitTag
         var hash = GitVersion
 
-        return  applicationName + " - " + tag + " - " + hash
+        var title = applicationName + " - " + tag
+        if(tag.includes("continuous")) {
```

Comment:
This tag 'continuous' will not exist on local development. Also, the tag will never be "" with the current `git describe` command. (and if it did, the application version would say pingviewer - No Tags! without any hash).

I've pushed some commits to handle this.

Reply:
We are not going to deploy with "local development".
git describe can fail if you are running a old git version.
The best way to fix the case where you don't have a tag is to use regex to find the release type.